GameData/RealismOverhaul/Engine_Configs/Veronique_Config.cfg

```diff
@@ -0,0 +1,346 @@
+				amount = 1
+			}
+
+			PROPELLANT
+			{
+				name = Kerosene
+				ratio = 0.3971
+				DrawGauge = True
+			}
+
+			PROPELLANT
+			{
+				name = IRFNA-III
+				ratio = 0.6029
+			}
+
+			atmosphereCurve
+			{
+				key = 0 200
+				key = 1 180
+			}
+		}
+
+		CONFIG
+		{
+			name = Veronique
+			description = Production Engine
+			minThrust = 44.4
+			maxThrust = 44.4
+			heatProduction = 100
+			massMult = 1.0
+			
+			ullage = True
+			pressureFed = False
+			ignitions = 1
+
+			IGNITOR_RESOURCE
+			{
+				name = ElectricCharge
+				amount = 0.5
+			}
+
+			IGNITOR_RESOURCE
+			{
+				name = Tonka250
+				amount = 1
+			}
+
+			PROPELLANT
+			{
+				name = Kerosene
+				ratio = 0.3971
+				DrawGauge = True
+			}
+
+			PROPELLANT
+			{
+				name = IRFNA-III
+				ratio = 0.6029
+			}
+
+			atmosphereCurve
+			{
+				key = 0 200
+				key = 1 180
+			}
+		}
+
+		CONFIG
+		{
+			name = VeroniqueAGI
+			description = Turpentine fuel for increased performance
+			minThrust = 43.6
+			maxThrust = 43.6
+			heatProduction = 100
+			massMult = 1.0
+			
+			ullage = True
+			pressureFed = False
+			ignitions = 1
+
+			IGNITOR_RESOURCE
+			{
+				name = ElectricCharge
+				amount = 0.5
+			}
+
+			IGNITOR_RESOURCE
+			{
+				name = Tonka250
+				amount = 1
+			}
+
+			PROPELLANT
+			{
+				name = Turpentine
+				ratio = 0.3464
+				DrawGauge = True
+			}
+
+			PROPELLANT
+			{
+				name = IWFNA
+				ratio = 0.6536
+			}
+
+			atmosphereCurve
+			{
+				key = 0 220
+				key = 1 202
+			}
+		}
+
+		CONFIG
+		{
+			name = Veronique61
+			description = Uprated version
+			minThrust = 65.3
+			maxThrust = 65.3
+			heatProduction = 100
+			massMult = 1.0
+			
+			ullage = True
+			pressureFed = False
+			ignitions = 1
+
+			IGNITOR_RESOURCE
+			{
+				name = ElectricCharge
+				amount = 0.5
+			}
+
+			IGNITOR_RESOURCE
+			{
+				name = Tonka250
+				amount = 1
+			}
+
+			PROPELLANT
+			{
+				name = Turpentine
+				ratio = 0.3464
+				DrawGauge = True
+			}
+
+			PROPELLANT
+			{
+				name = IWFNA
+				ratio = 0.6536
+			}
+
+			atmosphereCurve
+			{
+				key = 0 220
+				key = 1 202
+			}
+		}
+	}
+
+	!MODULE[ModuleAlternator]{}
+
+	!RESOURCE,*{}
+
+	RESOURCE
+	{
+		name = Tonka250
+		amount = 1
+		maxAmount = 1
+	}
+}
+
+@PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[VeroniqueR]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
+{
+	TESTFLIGHT
+	{
+		//7 successes, 1 failure
+		name = VeroniqueR
+		ratedBurnTime = 6.5
+		ignitionReliabilityStart = 0.85
+		ignitionReliabilityEnd = 0.97
+		cycleReliabilityStart = 0.875000
+		cycleReliabilityEnd = 0.975000
+
+	}
+}
+@PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[Veronique]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
+{
+	TESTFLIGHT
+	{
+		//35 successes, 19 failures
+		name = Veronique
+		ratedBurnTime = 45
+		ignitionReliabilityStart = 0.85
+		ignitionReliabilityEnd = 0.97
+		cycleReliabilityStart = 0.648148
+		cycleReliabilityEnd = 0.929630
+		techTransfer = VeroniqueR:50
```

Comment:
How do we derive the cycleReliabiltiyEnd value?  93% for a rocket that started with 65% seems high.  Do we have something like a running record of success/fail, and look at just the last eg 25% of launches for an end-reliability number?

Reply:
All reliability numbers were taken from a spreadsheet created by Agathorn, using real data.